client/layout/index.jsx

```diff
@@ -110,7 +110,7 @@ module.exports = React.createClass( {
 					</Welcome>
 					<InviteMessage sites={ this.props.sites }/>
 					<EmailVerificationNotice user={ this.props.user } />
-					<NoticesList id="notices" notices={ notices.list } forcePinned={ 'post' === this.state.section } />
```

Comment:
We need both of these components alive for now
Moving from old notices to new will take A LOT of time, because it involves connecting every component that sends a notice and is impossible to do in one PR
So I propose for the time beeing not touching `notices-list` and introducing them side by side.